src/logic/operate.js

```diff
@@ -4,11 +4,15 @@ const operate = (numberOne, numberTwo, operation) => {
   const one = Big(numberOne || 0);
   const two = Big(numberTwo || 0);
   let result;
-  if (operation === '+') result = one.plus(two);
-  if (operation === '-') result = one.minus(two);
-  if (operation === 'x') result = one.times(two);
-  if (operation === '÷') result = one.div(two);
-  if (operation === '%') result = one.times(two.div(100));
+  try {
+    if (operation === '+') result = one.plus(two);
+    if (operation === '-') result = one.minus(two);
+    if (operation === 'x') result = one.times(two);
+    if (operation === '÷') result = one.div(two);
+    if (operation === '%') result = one.times(two.div(100));
+  } catch (error) {
+    result = Big(0);
+  }
```

Comment:
You have just muted or stopped the operation. When we want to do a division by zero, we see nothing when we press zero. 
Make the zero show, then display an error message on display screen:
something like "division by zero error"

Reply:
Nice suggestion, I'll implement it.